### DISCUSSION TITLE
RegionStats for other species

### DISCUSSION THREAD

**Discussion by khanthanan** [stuart-lab/signac#1190] [Q&A] [answered]
Hi, 
This is a continuation of issue #1188. I managed to get the annotations for macaque, but I am having an issue with the RegionStats function. I have to specify a genome and I'm not sure how to go with it. I got the annotations from the gtf file, but I'm not sure how to get the genome. I believe this might not be directly related to Signac, but I would highly appreciate it if you could help me with a way to fix it. 

At the moment, I tried the following.

`library("BSgenome.Mmulatta.UCSC.rheMac3")`
`data <- RegionStats(data, genome =BSgenome.Mmulatta.UCSC.rheMac3)`

![image](https://user-images.githubusercontent.com/89409347/185115184-42b82875-59bc-4961-993d-def2f04c9331.png)

Thank you

**Comment by khanthanan** [comment, marked as answer]
I finally managed to get it fixed and wanted to share it with you. It was not an issue related to the genome or the annotation database, as I first thought. Since my dataset is relatively huge, I saved it as H5Seurat at the end of different steps and loaded it from there. One instance was after creating a peak assay using macs2_counts. I got this error when I loaded the Seurat object from there and called RegionStats. 

So as a solution, I saved both macs2_counts and the Seurat object and created the peak assay again before calling RegionStats. It worked without any errors. So something may be missing when it is saved as a Seurat object. 

I hope this helps. 

Best,

**Reply by HelloWorldLTY** [reply]
Hi, so based on my understanding, your solution is save peaks and rna seperately, then when you need to run regionStats, you can load them based on LoadH5seurat and then combine them together? Is it correct? Thanks.
